misc/l10n_testing.md

```diff
@@ -14,44 +14,110 @@ Each Mozilla project has its own process for taking your translations and making
+
+![Variable bug](/assets/images/l10n_errors/variable_bug.png)
+
 ### Broken access keys
 
+Access keys are unusable when the same key is assigned to multiple functions within the same menu. Additionally, if the access key uses a letter not available in the label, it's displayed near the label. Multiple identical access keys in the same context (preference menu, toolbar menu) will require the user to press the letter twice to use it. Ideally, repetitions should be avoided, but that's not possible in some places, e.g. new preferences. You can find broken access keys in your Firefox localization by looking at the list of access key errors in [Transvision](https://transvision.mozfr.org/accesskeys/) for your locale.
+
+![Access key bug](/assets/images/l10n_errors/access_keys.png)
+
 ### Broken language preferences
 
-### Broken search plugins
+Users experience broken website language preferences in Firefox when the accept-lang preferences are left unchanged from the en-US default set of language preferences. This can lead to the user receiving web pages in a language they're unfamiliar with by default, even when the web page is localized into their native language. It can be fixed in toolkit/chrome/global/intl.properties in the accept-lang setting.
+
+### Broken searchplugins
 
-Website unreachable message.
+Users experiencing a broken searchplugin will receive a "website unreachable" message when they attempt to perform a search from their awesome bar or search bar. If you discover that a searchplugin is broken, please file a bug with the l10n-drivers.
 
 ### Broken hyperlinks
 
+Broken hyperlinks occur in a couple of different ways:
+1) When there are typos in the hyperlink markup, the markup is then exposed to users.
+2) When the hyperlink links to a site that is not obvious by the text it's linked to (e.g., linking to the wrong support article).
+These can only be discovered by using the website or software regularly and clicking each hyperlink available to ensure that they take the user to the intended website.
+
 ## What to do when you find an error while testing
 
-Congratulations! You've found a bug! No software is perfect and you're very likely to find bugs while you test your localization.
+Congratulations! You've found a bug! No software is perfect and you're very likely to find bugs while you test your localization. Once you've found a bug, you can either attempt to find the affected string in your community's Pontoon projects or you can [file a bug in Bugzilla](https://bugzilla.mozilla.org/enter_bug.cgi) for that language in Mozilla Localizations :: [your_locale]. Once a fix for that bug is landed, it's important to test again to be sure that the issue has been resolved.
```

Comment:
You can point the link directly to https://bugzilla.mozilla.org/enter_bug.cgi?product=Mozilla%20Localizations